### ISSUE TITLE
ap=0

### ISSUE THREAD

**Issue by qq11-maker** [ShoufaChen/DiffusionDet#100]
When I train my own data set, how do I solve the ap=0 problem for the training set and the test set? thank you
